main: issue regaring negative argument is now fixed

Amitava123 opened the pull request:
Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Mentioned the issue number that it fixes
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

Amitava123 committed:
Merge pull request #2 from BytesClub/master

closing issue #3

Amitava123 committed:
modifying line 32

adding codes to take input from the user

fixes: #3

signed off by Amitava Mitra <mitraamitava7@gmail.com>

Amitava123 committed:
line no 33

adding codes for the user to get the position of X or O.

fixes: #2

signed off by: Amitava Mirea <mitraamitava7@gmail.com>

Amitava123 committed:
shortlog: adding lines in main.c at line 32

adding codes so that players can input 'X' or 'O' in the game.

fixes: #2

signed-off by: Amitava123<mitraamitava7@gmail.com>

Amitava123 committed:
shortlog: main.c

Fixing minor bug regarding negative argument passsing.

fixes: #13

signed-off by: Amitava123<mitraamitava7@gmail.com>

Amitava123 committed:
shortlog: issue fixed in line 21

the minor bug regarding negative argument is fixed.

fixes: #13

signed-off by:Amitava123<mitraamitava7@gmail.com>

0xTheProDev commented:
**argv[1]** is a String or a sequence of characters terminated by _NUL ('\0')_
The comparison operator **<** expects numeric value in each side, not array or string!

RudraNilBasu commented on main.c:
```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
 		printf("Usage: %s n\n", argv[0]);
 		return 1;
 	}
+	if(argv[1]<0){
```
A space between the ")" and "{"

https://bytesclub.github.io/docs/coding_style.pdf

RudraNilBasu commented on main.c:
```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
 		clear();
-		show(n, &n);
+                printf("\nPlayer 1: Enter Postion:\nRow Number: ");
+                row=getchar();
```
A space before and after a an operator

RudraNilBasu commented on main.c:
```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
+                row=getchar();
+                printf("\nColumn Number: ");
+                col=getchar();
+		ARR[row][col]='X';
```
Same space thing. + The name of the array needs to be changed. 
There might be a new issue targeting just on the coding structure

RudraNilBasu reviewed (commented):
Besides the comments added by @Progyan1997 I have also added a few comments on the coding structure.

Amitava123 committed:
main.c: line 21 adding *

modifying a small mistake in line no 21

fixes: #13

signed-off by: Amitava123<mitraamitava7@gmail.com>

0xTheProDev commented:
The issue #13 has already been resolved. The PR is closing!
